controllers/atlasschema_controller.go

```diff
@@ -261,6 +268,24 @@ func (r *AtlasSchemaReconciler) watch(sc *dbv1alpha1.AtlasSchema) {
 	}
 }
 
+// Clean up any resources created by the controller
+func (r *AtlasSchemaReconciler) cleanUp(ctx context.Context, sc *dbv1alpha1.AtlasSchema) {
+	pods := &corev1.PodList{}
+	if err := r.List(ctx, pods, client.MatchingLabels(map[string]string{
+		"app.kubernetes.io/instance": sc.Name + devDBSuffix,
+	})); err != nil {
+		r.recorder.Eventf(sc, corev1.EventTypeWarning, "CleanUpDevDB", "Error listing devDB pods: %v", err)
+	}
+	for _, p := range pods.Items {
+		err := r.Delete(ctx, &p)
+		if err != nil {
+			r.recorder.Eventf(sc, corev1.EventTypeWarning, "CleanUpDevDB", "Error deleting devDB pod %s: %v", p.Name, err)
+		}
+	}
+
+	r.recorder.Eventf(sc, corev1.EventTypeNormal, "CleanUpDevDB", "Recreated devDB pods")
```

Comment:
```suggestion
```
This is just part of normal operation, so I wouldn't spam the event log with it. If everything is fine, we should log one event at the end of processing the reconciliation.